index.html

```diff
@@ -11,11 +11,78 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+    <body>
```

Comment:
This is good you have indented the body tag by one tab. You should indent each child tag by one tab. For this file I would advise indenting the head tags as they are also children of the html tag.